component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/elk/ELKCounterMetric.java

```diff
@@ -58,11 +61,26 @@ public String getName() {
 
     @Override
     public MetricSchema getSchema() {
-        return null;
+        return schema;
     }
 
     @Override
     public MetricEventBuilder getEventBuilder() {
-        return new ELKMetricEventBuilder();
+
+        switch (schema) {
+            case RESPONSE:
+            default:
+                return new ELKMetricEventBuilder(
+                        GenericInputValidator.getInstance().getEventProperties(MetricSchema.ELK_RESPONSE));
+            case ERROR:
+                return new ELKMetricEventBuilder(
+                        GenericInputValidator.getInstance().getEventProperties(MetricSchema.ELK_ERROR));
+            case CHOREO_RESPONSE:
+                return new ELKMetricEventBuilder(
+                        GenericInputValidator.getInstance().getEventProperties(MetricSchema.CHOREO_RESPONSE));
+            case CHOREO_ERROR:
+                return new ELKMetricEventBuilder(
+                        GenericInputValidator.getInstance().getEventProperties(MetricSchema.CHOREO_ERROR));
```

Comment:
ELK report will not used my choreo CC. So we can remove this

Reply:
Fixed with https://github.com/wso2/apim-analytics-publisher/pull/96/commits/b1dfbfdcbd91651aebfef43686335037cc69b5a6